README.md

```diff
@@ -46,6 +46,7 @@ The AWS S3 build cache implementation requires two configuration options:
 | ----------------- | ----------- |
 | region | The AWS region the S3 bucket is located in. |
 | bucket | The name of the AWS S3 bucket where cache objects should be stored. |
+| reducedRedundancy | Whether or not to use [reduced redundancy](https://aws.amazon.com/s3/reduced-redundancy/). Enabled by default. |
```

Comment:
> The AWS S3 build cache implementation requires two configuration options (line 43)

This is no longer valid. What do you think about adding a new column "Default Value" and mentioning that all parameters which do not have a default are mandatory?